dashboard/src/actions/charts.ts

```diff
@@ -118,12 +118,17 @@ export function deployChart(
   releaseName: string,
   namespace: string,
   values?: string,
+  resourceVersion?: string,
```

Comment:
`resourceVersion` is an optional parameter that signals that we're trying to upgrade an existing release